etc/profile-a-l/imv.profile

```diff
@@ -0,0 +1,40 @@
+# Persistent local customizations
+include imv.local
+# Persistent global definitions
+include globals.local
+
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+include whitelist-runuser-common.inc
+
+apparmor
+caps.drop all
+net none
+nodvd
+noinput
+nonewprivs
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+shell none
```

Comment:
`tracelog`?

Reply:
Why?

Reply:
Because we add it to every profile where it works.